packaging/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -223,6 +223,47 @@ def test_plugin_honors_admin_only_config_option_when_rodsusers_manipulate_metada
         # Clean up.
         self.rods.assert_icommand(['imeta', 'rm', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
 
+    def test_plugin_disallows_rmw(self):
+        config = IrodsConfig()
+
+        # Set JSON configuration for the root collection.
+        root_coll = os.path.join('/', self.admin.zone_name)
+        json_config = json.dumps({
+            'prefixes': ['irods::'],
+            'admin_only': True
+        })
+        self.rods.assert_icommand(['imeta', 'set', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
+
+        try:
+            with lib.file_backed_up(config.server_config_path):
+                self.enable_rule_engine_plugin(config)
+
+                self.user.assert_icommand(['imeta', 'rmw', '-C', '%', '%', 'v'], 'STDERR', ['SYS_NOT_ALLOWED'])
```

Comment:
i think we're missing a collection name?

Reply:
rmw requires a specific targeted name, and then the wildcards can be in the AVU.